Dockerfile

```diff
@@ -16,12 +16,15 @@ ENV QUIPUCORDS_LOG_LEVEL=INFO
 
 COPY scripts/dnf /usr/local/bin/dnf
 RUN dnf install \
+        gcc \
         git \
         glibc-langpack-en \
         jq \
         make \
         openssh-clients \
+        postgresql-devel \
         python39 \
+        python39-devel \
```

Comment:
totally optional: since these new dependencies are required only for building, they could be uninstalled after we pip install and (presumably) compile everything we need.

we do that downstream:  
- https://pkgs.devel.redhat.com/cgit/containers/discovery-server/tree/Dockerfile?h=discovery-1.1-rhel-8#n10
- https://pkgs.devel.redhat.com/cgit/containers/discovery-server/tree/Dockerfile?h=discovery-1.1-rhel-8#n49